### PR TITLE
Changing Claims Title and Description

### PR DESCRIPTION
**Pull request by bastosmichael**
Quick nomenclature and description change for Claims

### CODE REVIEW

File: src/apiDefs.ts [hunk 1]
```diff
@@ -29,9 +29,9 @@ export const apiDefs : IApiCategories = {
     apiKey: true,
     apis: [
       {
-        name: 'Benefits Intake',
-        shortDescription: 'Submit claims',
-        urlFragment: 'benefits',
+        name: 'Claims',
```
Comment (juliaelman) [inline src/apiDefs.ts]: s Claims/Benefits Claims

File: src/apiDefs.ts [hunk 2]
```diff
@@ -29,9 +29,9 @@ export const apiDefs : IApiCategories = {
...
+        shortDescription: 'Submit and Track claims',
```
Comment (juliaelman) [inline src/apiDefs.ts]: s Track/track

File: src/apiDefs.ts (hunk 1 above)
Comment (bastosmichael) [reply src/apiDefs.ts]: Done

File: src/apiDefs.ts (hunk 2 above)
Comment (bastosmichael) [reply src/apiDefs.ts]: Done

**Pull request merged**